Orange/widgets/visualize/owboxplot.py

```diff
@@ -972,6 +1027,44 @@ def send_report(self):
         if text:
             self.report_caption(text)
 
+    class Label(QGraphicsSimpleTextItem):
+        """Boxplot Label with settable maxWidth"""
```

Comment:
Should we move something like this to the gui module, so it can be reused elsewhere?

Reply:
When anyone else needs a label like this, it can be moved and imported from there. Until then, I see no point in further polluting the gui namespace.